src/test/java/com/mongodb/jdbc/MongoDatabaseMetaDataTest.java

```diff
@@ -6,25 +6,34 @@
 import java.sql.ResultSet;
 import java.sql.ResultSetMetaData;
 import java.sql.SQLException;
+
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-class MongoDatabaseMetaDataTest extends MySQLMock {
-    private static DatabaseMetaData databaseMetaData;
+abstract class MongoDatabaseMetaDataTest {
+    protected DatabaseMetaData databaseMetaData;
+
+    protected abstract DatabaseMetaData createDatabaseMetaData();
 
-    static {
-        databaseMetaData = new MySQLDatabaseMetaData(null);
+    @BeforeEach
```

Comment:
This is odd, if you want to share the stacktrace offline, I can take a quick look. Here, it's probably not too much of a problem to initialize it for each test since it's not too costly. It could be hiding a potential issue with the metadata code though.